great_expectations/expectations/metrics/column_aggregate_metrics/column_histogram.py

```diff
@@ -69,10 +99,32 @@ def _sqlalchemy(
         column = accessor_domain_kwargs["column"]
         bins = metric_value_kwargs["bins"]
 
+        """return a list of counts corresponding to bins"""
         case_conditions = []
         idx = 0
         if isinstance(bins, np.ndarray):
             bins = bins.tolist()
+        elif isinstance(bins, int):
+            sqlalchemy_engine = execution_engine.engine
+            query: Select = (
+                sa.select(sa.column(column))
+                .distinct()
+                .where(sa.column(column) != None)
+                .order_by(sa.column(column).asc())
+                .select_from(selectable)
+            )
+            try:
+                rows: List[Row] = sqlalchemy_engine.execute(query).fetchall()
+                row: Row
+                column_values: np.ndarray = np.asarray([row[0] for row in rows])
+                bins = np.histogram_bin_edges(column_values, bins=bins)
+                bins = bins.tolist()
```

Comment:
Is this going to work on larger data? Won't this load all data to the node running GE? Can we offload instead to the database i.e. put this computation in the query?